_data/tools.yml

```diff
@@ -3,13 +3,16 @@ it:
     title: Codice sorgente
     subtitle: Sul nostro GitHub Italia, sviluppiamo con un processo aperto gli SDK e gli esempi di supporto dei progetti, e gestiamo anche le segnalazioni dei problemi.
     icon: list
+    url: https://github.com/italia/developers.italia.it
```

Comment:
Puoi cambiare questo in GitHub.com/italia? Vorrei linkare l'intera organizzazione, non il singolo repo.